docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,244 @@
+.. _guide_scaleway:
+
+***************************
+Using Scaleway with Ansible
+***************************
+
+.. _scaleway_introduction:
+
+Introduction
+============
+
+`Scaleway <https://scaleway.com>`_ is a cloud provider supported by Ansible, version 2.6 or higher via a dynamic inventory plugin and modules.
+Those modules are:
+
+- :ref:`scaleway_sshkey`: adds a public SSH key from a file or value to the Packet infrastructure. Every subsequently-created device will have this public key installed in .ssh/authorized_keys.
```

Comment:
To link to https://docs.ansible.com/ansible/latest/modules/scaleway_sshkey_module.html, the `:ref:` must use `scaleway_sshkey_module` instead of `scaleway_sshkey` - sorry if my earlier comment wasn't clear.